### PR TITLE
=doc #17329 Spelling normalization, typos fixed.

### PR DESCRIPTION
**Pull request by kukido**

### CODE REVIEW

File: akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala [hunk 1]
```diff
@@ -196,7 +196,7 @@ case class ValidationRejection(message: String, cause: Option[Throwable] = None)
  * 3. A TransformationRejection holding a function filtering out the MethodRejection
  *
  * so that in the end the RejectionHandler will only see one rejection (the ValidationRejection), because the
- * MethodRejection added by the ``get`` directive is cancelled by the ``put`` directive (since the HTTP method
+ * MethodRejection added by the ``get`` directive is canceled by the ``put`` directive (since the HTTP method
```
Comment (ktoso) [inline akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala]: ok, let's go with US english I guess.

File: akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala [hunk 2]
```diff
@@ -213,7 +213,7 @@ class Flow[-In, +Out, +Mat](delegate: scaladsl.Flow[In, Out, Mat]) extends Graph
    *
    * Make sure that the `Iterable` is immutable or at least not modified after
    * being used as an output sequence. Otherwise the stream may fail with
-   * `oncurrentModificationException` or other more subtle errors may occur.
+   * `ConcurrentModificationException` or other more subtle errors may occur.
```
Comment (ktoso) [inline akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala]: nice catch :)

**Comment by ktoso** [comment]
LGTM, thanks!

**Comment by ktoso** [comment]
Refs #17329

File: akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala (hunk 1 above)
Comment (kukido) [inline akka-http/src/main/scala/akka/http/scaladsl/server/Rejection.scala]: That's what was the ticket about :smile:

**Pull request merged**